course-material/interactive-queries-and-parameterization/introduction-to-ipywidgets.md

```diff
@@ -1,15 +1,376 @@
 ---
-jupyter:
-  jupytext:
-    text_representation:
-      extension: .md
-      format_name: markdown
-      format_version: '1.3'
-      jupytext_version: 1.14.5
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.6
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
 ---
 
-# Introduction to ipywidgets
+# Introduction to `ipywidgets`
 
+```{code-cell} ipython3
+:tags: [remove-cell]
 
+# this cell is hidden in the docs, only used to simulate
+# the getpass() call
+import ipywidgets as widgets
+from urllib.request import urlretrieve
+from zipfile import ZipFile
+import pandas as pd
+import os
```

Comment:
It's really confusing to have imports at the top if there are installation steps for these libraries at the bottom

Remove the imports from this section